packages/validator/test/inference.spec.js

```diff
@@ -93,3 +93,152 @@ test('use InferInvokedCapability', () =>
       result.product.toLocaleLowerCase()
     }
   })
+
+test('infers nb fields optional', () => {
+  capability({
+    can: 'test/nb',
+    with: DID.match({ method: 'key' }),
+    nb: {
+      msg: URI.match({ protocol: 'data:' }),
+    },
+    derives: (claim, proof) => {
+      /** @type {string} */
+      // @ts-expect-error - may be undenfined
```

Comment:
This typo occurs many times...

```suggestion
      // @ts-expect-error - may be undefined
```